src/rust/src/rustzcash.rs

```diff
@@ -352,23 +328,24 @@ fn priv_get_note(
     pk_d: *const [c_uchar; 32],
     value: u64,
     r: *const [c_uchar; 32],
```

Comment:
This parameter being called `r` is a bit confusing; consider renaming it to `rcm` here, and also in `librustzcash_sapling_compute_cmu` and `librustzcash_sapling_compute_nf`.